backend/icpy/agent/tools/doc_processor/pptx_handler.py

```
@@ -0,0 +1,277 @@
+                    row_data.append(cell.text.strip() if cell.text else "")
+                rows_data.append(row_data)
+            
+            if not rows_data:
+                return None
+            
+            # Use first row as headers
+            headers = rows_data[0] if rows_data else []
+            headers = [h if h else f"col_{i}" for i, h in enumerate(headers)]
+            
+            data = [
+                dict(zip(headers, row))
```

Ruff (0.14.14)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)